### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Baroka-wp**
- [ ] Create .gitignore
- [ ] Create linter-config
- [ ] Update README.md
- [ ] Create hello-world.html
- [ ] Create style.css
- [ ] Link style.css in .html file
- [ ] Add test.md to .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+- Major languages
+- Frameworks
+- Technologies used
+
+
+## Authors
+
+👤 **Author1**
+
```
Comment (ShadyShawkat) [inline README.md]: - [ ] [OPTIONAL] Please change this to be your name instead.

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+# Hello-world
+
+> The goal here is to master all of the tools and best practices we learned about github flow
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```
Comment (ShadyShawkat) [inline README.md]: - [ ] [OPTIONAL] Kindly change the list of technologies that you used to build this project here. (HTML, CSS, linters, ...etc)

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Baroka-wp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Baroka-wp) [reply README.md]: @ShadyShawkat thank you for your response.

**Pull request merged**